### DISCUSSION TITLE
OpenTCS integration with Unity 3D

### DISCUSSION THREAD

**Discussion by sahithiakku** [openTCS/opentcs#53] [Q&A]
Would you kindly provide me with any supporting materials or instruction manuals on how to combine OpenTCS with Unity 3D for simulation?

**Comment by swltr** [comment]
See the [section in the Developer's Guide](https://www.opentcs.org/docs/5.6/developer/developers-guide/opentcs-developers-guide.html#section-kernel-api) and the [web API specification](https://www.opentcs.org/docs/5.6/developer/service-web-api-v1/index.html?docExpansion=list).
